index.js

```diff
@@ -185,6 +186,10 @@ class Dynamodb extends Datastore {
             });
         }
 
+        if (config.sort) {
+            scanner = scanner[config.sort]();
```

Comment:
My only concern is if someone passes in an invalid option for `config.sort`, this fails. Since this is a boolean choice, can we just do something more defensive like:

```
scanner = (config.sort === 'ascending') ? scanner.ascending() : scanner.descending();
```

It'll be more tolerable to silly user input.